DependencyInjection/MonologExtension.php

```diff
@@ -520,6 +520,9 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
             if ($handler['ident']) {
                 $definition->addArgument($handler['ident']);
             }
+            if (isset($handler['rfc'])) {
```

Comment:
Hmm what if `ident` is null/false and this value is set, I believe, you'll end up with `$handler['rfc']` injected in `$ident` argument.
Could you add a test to assert that scenario won't happend?

Reply:
Seems like everything is ok

Reply:
nop.
1. it does not work
2. you didn't added test for it

```
class MonologExtensionTest extends DependencyInjectionTest
{
    public function testSyslog()
    {
        $container = new ContainerBuilder();

        $loader = new MonologExtension();
        $configs = [
            [
                'handlers' => [
                    'syslogudp' => [
                        'type' => 'syslogudp',
                        'host' => '127.0.0.1',
                        'port' => 514,
                        'facility' => 'USER',
                        'ident' => null,
                        'level' => 'ERROR',
                        'rfc' => SyslogUdpHandler::RFC5424,
                    ]
                ]
            ]
        ];
        $expectedArguments = [
            '127.0.0.1', // $host
            514, // $port
            'USER', // $facility
            400, // $level
            true, // $bubble
            'php', // $ident,
            1, // $rfc
        ];
        $loader->load($configs, $container);

        $definition = $container->getDefinition('monolog.handler.syslogudp');
        $this->assertDICConstructorArguments($definition, $expectedArguments);
    }
}
```

> 1) Symfony\Bundle\MonologBundle\Tests\DependencyInjection\MonologExtensionTest::testSyslog
> Expected and actual DIC Service constructor arguments of definition 'Monolog\Handler\SyslogUdpHandler' don't match.
> Failed asserting that two arrays are equal.
```diff
--- Expected
+++ Actual
@@ @@
     2 => 'USER'
     3 => 400
     4 => true
-    5 => 'php'
-    6 => 1
+    5 => 1
 )
```

Reply:
good catch

Reply:
Yep, my bad. 

https://github.com/Seldaek/monolog/blob/main/src/Monolog/Handler/SyslogUdpHandler.php#L49 here i see that ident has default value and it's a string

Maybe we should make this argument required with the same default value as into SyslogUdpHandler?

Reply:
I mean that we can change this `->scalarNode('ident')->defaultFalse()->end() // syslog and syslogudp` to `->scalarNode('ident')->defaultValue('php')->end() // syslog and syslogudp` and change this

```

Reply:
IMHO `->scalarNode('ident')->defaultValue('php')->end()` is simpler

Reply:
yep, it's simpler, but if we only change to `->scalarNode('ident')->defaultValue('php')->end()` than [problem](https://github.com/symfony/monolog-bundle/pull/343#discussion_r601499445) still happens